app/build.gradle.kts

```diff
@@ -135,7 +135,6 @@ dependencies {
     val retrofit_version = "2.7.1"
     implementation("com.squareup.retrofit2:retrofit:$retrofit_version")
     implementation("com.squareup.retrofit2:converter-gson:$retrofit_version")
-    implementation("com.squareup.retrofit2:adapter-rxjava:$retrofit_version")
```

Comment:
This is still used in the GithubService so it can't be removed. I'm not familiar with how this call works for that so if you think it can be removed from the service, I'd appreciate it.

Reply:
Should be good now.  Retrofit made it straighforward

https://github.com/square/retrofit/blob/master/CHANGELOG.md#version-260-2019-06-05